packages/core/src/dynamoDb/commands/searchDynamoDbTables.ts

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import fs from '../../shared/fs/fs'
+import DynamoDB from 'aws-sdk/clients/dynamodb'
+import { Wizard } from '../../shared/wizards/wizard'
+import { telemetry } from '../../shared/telemetry/telemetry'
+import { DataQuickPickItem } from '../../shared/ui/pickerPrompter'
+import { DynamoDbClient } from '../../shared/clients/dynamoDbClient'
+import { toArrayAsync } from '../../shared/utilities/collectionUtils'
+import { CancellationError } from '../../shared/utilities/timeoutUtils'
+import { RegionSubmenu, RegionSubmenuResponse } from '../../shared/ui/common/regionSubmenu'
+
+const localize = nls.loadMessageBundle()
+
+/**
+ * "Search DynamoDb Tables Command"
+ *
+ */
+export async function searchDynamoDbTables(source: string, dbData?: { regionName: string }): Promise<void> {
+    await telemetry.dynamodb_openTable.run(async span => {
+        const wizard = new SearchDynamoDbTablesWizard(dbData)
+        span.record({ dynamoDbResourceType: 'table', source: source })
+        const response = await wizard.run()
+        if (!response) {
+            throw new CancellationError('user')
+        }
+        const tableOutput = await getItemsFromTable(response.submenuResponse.region, response.submenuResponse.data)
+
+        const output = `${JSON.stringify(tableOutput.Items, undefined, 4)}\n`
+        const filePath = process.env['HOME'] + '/Documents/dynamoDbItems.json'
```

Comment:
note for anyone else reading, this is temporary. The next PR will probably be the webview one